core/src/com/cloud/storage/template/VhdProcessor.java

```diff
@@ -45,10 +49,12 @@
     private static final Logger s_logger = Logger.getLogger(VhdProcessor.class);
     StorageLayer _storage;
     private int vhdFooterSize = 512;
+    private int vhdCookieOffset = 8;
```

Comment:
Hey Syed - Can you give me a bit of background on this file needing to be changed? Is this a bug fix that is being integrated into the work of this PR or is this file being changed because of the new functionality of this PR? Thanks!